### PR TITLE
[Testing] Enable snapshot tests on Catalyst

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Enable snapshot tests on Catalyst.

### CODE REVIEW

File: src/Controls/tests/TestCases.Mac.Tests/snapshots/mac/ImageSourceInitializesCorrectly.png
Comment (jsuarezruiz) [inline src/Controls/tests/TestCases.Mac.Tests/snapshots/mac/ImageSourceInitializesCorrectly.png:1]: Clearly, the behavior from the Buttons is incorrect, same as on iOS actually. Will be fixed by https://github.com/dotnet/maui/pull/25373
After merge that PR, this snapshot must be updated.

**Review by jfversluis — CHANGES_REQUESTED** [review]
Has conflicts now, sorry!

**Review by jfversluis — APPROVED** [review]

**Pull request merged**